### ISSUE TITLE
How to train own's data images？

### ISSUE THREAD

**Issue by ZhuJD-China** [abc-125/segad#2]

**Comment by IceIce1ce** [comment]
+1

**Comment by baraa-m** [comment]
Thank you for this interesting work! 
I have the same question , how we can train the model on our own dataset

**Comment by abc-125** [comment]
Thank you @baraa-m!

Here is the instruction how to train it on your own images:
1. Train the anomaly detection model of your choice (you can use [anomalib](https://github.com/openvinotoolkit/anomalib)) on part of your training data
2. Take the rest of your training data and:
	2.1. Get segmentation masks (can be produced by SAM or a trained segmentation model; see [paper](https://arxiv.org/abs/2405.04953) for more details)
	2.2 Use the trained anomaly detection model from Step 1 to produce anomaly maps
3. Use data from Step 2 to train SegAD; code in this repo utilizes pandas data frames to access filenames and labels

**Comment by mohandeeprdori** [comment]
How do I train this on VAD dataset?

**Comment by abc-125** [comment]
@mohandeeprdori The same as described above, I will try to add the script later. You can also find more details in the paper.
